File: source/Nebb.Nss.2022.Aws.Model/Entities/MediaFiles.cs
using Newtonsoft.Json;

namespace Nebb.Nss._2022.Aws.Model.Entities
{
    public class MediaFiles
    {
        [JsonProperty(PropertyName = "id")]
        public string Id { get; set; }
        public string Name { get; set; }
        public string Type { get; set; }
        public int Size { get; set; }
        public string FileUrl { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime LastModifiedAt { get; set; }

        public string ThumbnailUrl { get; set; }
        public bool IsDeleted { get; set; } = false;
    }
}

File: source/Nebb.Nss.2022.Aws.UI/NebbSummerSchool/src/app/core/models/MediaFiles.ts
export class MediaFile {
  id: string;
  name: string;
  type: string;
  size!: number;
  fileUrl: string;
  createdAt!: Date;
  lastModifiedAt!: Date;
  thumbnailUrl!: string;
  isDeleted: boolean;

  constructor() {
    this.id = '';
    this.name = '';
    this.type = '';
    this.fileUrl = '';
    this.isDeleted = false;
  }
}
